ydb/docs/en/core/integrations/goose.md

```diff
@@ -0,0 +1,128 @@
+# Versioning data schema and migration in YDB using "goose"
+
+## Introduction
+
+Goose is an open-source tool that helps version the data schema in the database and manage migrations. Goose supports many different databases, including YDB. Goose uses migration files and stores the state of migrations directly in the database in a special table.
+
+## Installation goose
+
+Goose installation options are described in [documentation](https://github.com/pressly/goose/blob/master/README.md#install).
+
+## Launch arguments goose
+
+The `goose` utility is called with the command:
+
+```
+$ goose <DB> <DSN> <COMMAND> <COMMAND_ARGUMENTS>
+```
+
+Where:
+- `<DB>` - database engine, in the case of YDB you should write `goose ydb`
+- `<DSN>` - database connection string.
+- `<COMMAND>` - the command to be executed. A complete list of commands is available in the built-in help (`goose help`).
+- `<COMMAND_ARGUMENTS>` - command arguments.
+
+## YDB connection string
+
+To connect to YDB you should use a connection string like
+
+```
+<protocol>://<host>:<port>/<database_path>?go_query_mode=scripting&go_fake_tx=scripting&go_query_bind=declare,numeric
+```
+
+Where:
+- `<protocol>` - connection protocol (`grpc` for an unsecured connection or `grpcs` for a secure (`TLS`) connection). The secure connection (with `TLS`) requires certificates. You should declare certificates like this: `export YDB_SSL_ROOT_CERTIFICATES_FILE=/path/to/ydb/certs/CA.pem`.
```

Comment:
needs a link to documentation on how to obtain these certificates

Reply:
It is not simple for describe way for obtain certificates, but it is noisy for article
For example: Yandex Cloud CA.pem have one path, docker local ydb have another path (mount ydb_certs), bare-metal on-premice installation have third path (which devops maked handle using certbot or another CA)